packages/react/src/themes/teams/components/Icon/svg/icons/index.ts

```diff
@@ -12,6 +12,8 @@ import bullets from './bullets'
 import calendar from './calendar'
 import call from './call'
 import callParking from './callParking'
+import callControlRelease from './callControlRelease'
```

Comment:
lets maintain alphabetical order, to simplify search scenarios

Reply:
I am keeping release and request as a group (viewer) as they are most closely related. The other call-control-xxx icons are a different group (sender)

Reply:
I have grouped them by concepts as opposed to lexical order.

Reply:
The existing list is already out of order. Do you want me to fix it?

Reply:
no, lets stick to the order pattern already established -  thank you for clarification!